image_width: 640
image_height: 480
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [771.33999,   0.     , 350.41938,
          0.     , 771.18321, 296.66075,
          0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.202376, 0.520275, 0.000945, -0.001841, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
        0., 1., 0.,
        0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [763.6826 ,   0.     , 348.14486,   0.     ,
          0.     , 765.71012, 296.26671,   0.     ,
          0.     ,   0.     ,   1.     ,   0.     ]